src/libstd/path.rs

```diff
@@ -1248,6 +1248,13 @@ impl Path {
     /// use std::path::Path;
     ///
     /// Path::new("foo.txt");
+    ///
+    /// // Strings work too
+    /// let s = String::from("bar.txt");
+    /// let p = Path::new(&s);
+    ///
+    /// // As do other Paths
```

Comment:
`Path`s should have the `s around it, and a period or a colon, as well as the newlines and triple backticks